libs/ledger-live-common/src/families/aptos/estimateMaxSpendable.test.ts

```diff
@@ -4,16 +4,18 @@ import estimateMaxSpendable from "./estimateMaxSpendable";
 import BigNumber from "bignumber.js";
 
 jest.mock("./getFeesForTransaction", () => ({
-  getEstimatedGas: jest.fn(() => ({
-    fees: new BigNumber(0),
-    estimate: {
-      maxGasAmount: 1,
-      gasUnitPrice: 2,
-      sequenceNumber: "",
-      expirationTimestampSecs: "",
-    },
-    errors: {},
-  })),
+  getEstimatedGas: jest.fn(() =>
+    Promise.resolve({
+      fees: new BigNumber(0),
+      estimate: {
+        maxGasAmount: 1,
+        gasUnitPrice: 2,
+        sequenceNumber: "",
+        expirationTimestampSecs: "",
+      },
+      errors: {},
+    }),
+  ),
```

Comment:
I think you can drop the `Promise.resolve` here.

```suggestion
getEstimatedGas: jest.fn(() => ({
    fees: new BigNumber(0),
    estimate: {
      maxGasAmount: 1,
      gasUnitPrice: 2,
      sequenceNumber: "",
      expirationTimestampSecs: "",
    },
    errors: {},
  })),
```